src/styles/variables.css

```diff
@@ -52,6 +52,17 @@
     --amp-colors-button-text: var(--amp-colors-white);
 
     /* input */
-    --amp-colors-input-hover: var(--amp-colors-bg-highlight);
```

Comment:
deleting these may cause breaking or unexpected change for users that may have custom override this value. (maybe unlikely if not adopted yet)